doc/developer-guides/hld/hld-APL_GVT-g.rst

```diff
@@ -320,8 +320,8 @@ guests. A GVT-g device model working with the ACRN hypervisor
 implements the policies of trap and passthrough. Each guest runs the
 native graphics driver and can directly access performance-critical
 resources: the Frame Buffer and Command Buffer, with resource
-partitioning (as presented later). To protect privileged resources—that
-is, the I/O registers and PTEs—corresponding accesses from the graphics
+partitioning (as presented later). To protect privileged resources, that
```

Comment:
If you don't like the em dashes, use parentheses. The current plethora of commas is confusing.